projects/1.19/assets/farmers-utils/farmersutils/lang/zh_cn.json

```
@@ -0,0 +1,58 @@
+{
+  "itemGroup.farmersutils": "储存乐事",
+
+  "block.farmersutils.apple_bag": "袋装苹果",
+  "block.farmersutils.brown_mushroom_bag": "袋装棕色蘑菇",
+  "block.farmersutils.cocoa_bean_bag": "袋装可可豆",
+  "block.farmersutils.crimson_fungus_bag": "袋装绯红菌",
+  "block.farmersutils.egg_box": "盒装鸡蛋",
+  "block.farmersutils.glow_berry_bag": "袋装发光浆果",
+  "block.farmersutils.golden_apple_bag": "袋装金苹果",
+  "block.farmersutils.kelp_block": "海带块",
+  "block.farmersutils.red_mushroom_bag": "袋装红色蘑菇",
+  "block.farmersutils.sugar_cane_bale": "甘蔗捆",
```

```suggestion
  "block.farmersutils.sugar_cane_bale": "甘蔗块",
```
`捆`这一称呼适用于1.19.3及以上版本